projects/1.16/assets/advanced-hook-launchers/adhooks/lang/zh_cn.json

```
@@ -1,24 +1,44 @@
-  "keybind.adhooks.unhook": "解钩",
-  "tooltip.adhooks.launcher.controls": "%1$s按住%2$s %3$s%1$s 以获取控制帮助",
-  "tooltip.adhooks.launcher.launch": "%1$s发射：%2$s",
-  "tooltip.adhooks.launcher.boostup_strength": "上推强度",
-  "tooltip.adhooks.launcher.reeling_speed": "卷取速度",
-  "tooltip.adhooks.launcher.rope_max_length": "绳子最大长度",
-  "tooltip.adhooks.launcher.rope_elasticity": "绳子弹性",
-  "tooltip.adhooks.launcher.hook_strength": "钩抓强度",
-  "entity.adhooks.hookshot": "钩抓",
-  "item.adhooks.web_launcher": "网钩抓发射器",
-  "item.adhooks.pudge_launcher": "胖钩抓发射器",
-  "item.adhooks.spear_launcher": "矛钩抓发射器",
-  "item.adhooks.prototype_launcher": "原型发射器",
-  "item.adhooks.web_hook": "网钩抓",
-  "item.adhooks.spear_hook": "矛钩抓",
-  "item.adhooks.pudge_hook": "胖钩抓",
-  "tooltip.adhooks.launcher.boost_up": "%1$s上推：当拉的时候，摁下 %2$s 或者 %3$s",
-  "tooltip.adhooks.launcher.more_info": "%1$shold%2$s%3$s%1$s了解更多信息",
-  "tooltip.adhooks.launcher.release": "%1$s释放：当摁住 %3$s 时摁 %2$s",
-  "tooltip.adhooks.launcher.loosen": "%1$s松开：%2$s（摁下）"
-}
\ No newline at end of file
+	"item.adhooks.spear_hook": "普通钩子",
+	"item.adhooks.pudge_hook": "生物抓取钩",
+	"item.adhooks.web_hook": "蛛网钩子",
+	"item.adhooks.prototype_launcher": "没有钩子的发射器",
+	"item.adhooks.spear_launcher": "普通钩爪",
+	"item.adhooks.pudge_launcher": "生物抓取器",
+	"item.adhooks.web_launcher": "蛛网钩爪",
+	"entity.adhooks.hookshot": "钩爪",
+	"key.categories.adhooks.main_hand": "钩爪发射器（主手）",
+	"key.adhooks.main_hand.launching": "发射",
+	"key.adhooks.main_hand.unhooking": "卷起绳索/脱钩",
+	"key.adhooks.main_hand.pulling": "拉绳索",
+	"key.adhooks.main_hand.loosening": "放线",
+	"key.adhooks.main_hand.jumping": "跳",
+	"key.categories.adhooks.off_hand": "钩爪发射器（副手）",
+	"key.adhooks.off_hand.launching": "发射",
+	"key.adhooks.off_hand.unhooking": "卷起绳索/脱钩",
+	"key.adhooks.off_hand.pulling": "拉绳索",
+	"key.adhooks.off_hand.loosening": "放绳索",
+	"key.adhooks.off_hand.jumping": "跳",
+	"tooltip.adhooks.launcher.hook_strength": "钩爪耐久",
+	"tooltip.adhooks.launcher.rope_elasticity": "绳索弹性",
+	"tooltip.adhooks.launcher.rope_max_length": "绳索最大长度",
+	"tooltip.adhooks.launcher.reeling_speed": "卷绳索速度",
+	"tooltip.adhooks.launcher.jump_boost_strength": "跳跃提升",
+	"tooltip.adhooks.launcher.tarzan_jump_strength": "大跳",
+	"tooltip.adhooks.launcher.launch": "%1$s发射:%2$s 按下 %1$s%3$s%2$s",
+	"tooltip.adhooks.launcher.reel_unhook": "%1$s卷起绳索/脱钩:%2$s 按下 %1$s%3$s%2$s",
+	"tooltip.adhooks.launcher.reel_unhook_same": "%1$s卷起绳索/脱钩:%2$s 在长按 %1$s%4$s 时，按下 %1$s%3$s%2$s",
+	"tooltip.adhooks.launcher.pull": "%1$s拉绳索:%2$s 按下或长按%1$s%3$s",
+	"tooltip.adhooks.launcher.loosen": "%1$s放绳索:%2$s 按下或长按 %1$s%3$s%2$s",
+	"tooltip.adhooks.launcher.loosen_same": "%1$s放绳索:%2$s 在长按 %1$s%4$s 时，按下或长按 %1$s%3$s%2$s",
+	"tooltip.adhooks.launcher.swing": "%1$s摆荡着前进:%2$s 用 %1$s%3$s%2$s 来制造动力",
+	"tooltip.adhooks.launcher.jump_boost": "%1$s跳跃提升:%2$s 在拉绳索时, 按下 %1$s%3$s%2$s 的同时长按 %1$s%4$s",
+	"tooltip.adhooks.launcher.tarzan_jump": "%1$s大跳:%2$s 在空中时，双击 %1$s%3$s%2$s",
```

```suggestion
	"tooltip.adhooks.launcher.tarzan_jump": "%1$s大跳:%2$s 在空中时，双击 %1$s%3$s%2$s",
```
两段跳（但这并不一定是在空中双击空格）或泰山跳（字面意思）